### PR TITLE
making web multiapi

### PR DESCRIPTION
**Pull request by zikalino**

### CODE REVIEW

File: specification/web/resource-manager/readme.md [hunk 1]
```diff
@@ -97,59 +97,709 @@ directive:
...
+- Microsoft.Web/stable/2018-02-01/ResourceHealthMetadata.json
+directive:
+  # suppress each RPC 3019 error
+- where: $.definitions.Identifier.properties
+  suppress: R3019
+  reason: It's an old API, will resolve in next API version
+  approved-by: "@ravbhatnagar"
+```
+
+### Tag: package-2018-02-only
```
Comment (MyronFanQiu) [inline specification/web/resource-manager/readme.md]: Hello. Looks like this tag is same with `package-2018-02`. Is that by design?

Comment (zikalino) [reply specification/web/resource-manager/readme.md]: yes

Comment (zikalino) [reply specification/web/resource-manager/readme.md]: wait

Comment (zikalino) [reply specification/web/resource-manager/readme.md]: yes, it's by design

File: specification/web/resource-manager/readme.md [hunk 2]
```diff
@@ -97,59 +97,709 @@ directive:
...
+- Microsoft.Web/stable/2016-03-01/Diagnostics.json
+- Microsoft.Web/stable/2016-03-01/Provider.json
+- Microsoft.Web/stable/2016-03-01/Recommendations.json
+- Microsoft.Web/stable/2016-03-01/ResourceHealthMetadata.json
+- Microsoft.Web/stable/2016-03-01/ResourceProvider.json
+- Microsoft.Web/stable/2016-08-01/WebApps.json
+- Microsoft.Web/stable/2016-09-01/AppServiceEnvironments.json
+- Microsoft.Web/stable/2016-09-01/AppServicePlans.json
+directive:
+  # suppress each RPC 3019 error
```
Comment (Juliehzl) [inline specification/web/resource-manager/readme.md]: Why are there so may R3019 error to be suppressed?

**Review by mmyyrroonn — APPROVED** [review]

**Pull request merged**